src/main/features/ccj/views/confirmation.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "layout.njk" %}
+{% from "includes/macros/form.njk" import submitButton, errorSummary %}
+{% from "includes/macros/support.njk" import supportWidget %}
+{% set title = t('CCJ requested') %}
+
+{% block content %}
+
+  <div class="grid-row">
+
+    <div class="govuk-box-highlight">
+      <h1 class="bold-large">{{ t('County Court Judgment requested') }}</h1>
+      {#
+      <p class="issue-receipt-download">
+        <a href="{{ judgmentDownloadPath }}">{{ t('Download judgment as pdf') }}</a>
+      </p>
+      #}
+    </div>
+
+    <h2 class="heading-medium">{{ t('What happens next') }}</h2>
+    <p>{{ t('The court has entered judgment in your favour, %s is no longer able to file a response.',
+      { postProcess: 'sprintf', sprintf: [ defendantName ]}) }}
+    </p>
+    <p>{{ t('You can ask the court to enforce the judgment if %s doesn’t pay as ordered by %s.',
+      { postProcess: 'sprintf', sprintf: [ defendantName, judgmentDeadline | date ]}) }}</p>
+
+    <p>{{ t('View your claim status on <a href="%s">your money claims account</a>.',
+      {
+        postProcess: 'sprintf',
+        sprintf: ['/dashboard']
```

Comment:
why don't we use "/dashboard" as a value of href attr?

Reply:
not to translate it maybe? I copied it from the response confirmation page

Reply:
ok